### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by vijaytembugade**
-  Added Introduction and Colors.
-  Added Typography
-  Added titles icons and Some animations.
- Css Refactoring

### CODE REVIEW

**Review by manojkumar8400 — APPROVED** [review]
looks good

**Review by horsemaker — APPROVED** [review]
looks fine to me! 👌

File: Components/Badge/badge.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import url("../Avatar/avatar.css");
+
 /* normal badge  */
 .badge {
   padding: 0rem 0.2rem;
```
Comment (BelideNikhil) [inline Components/Badge/badge.css:4]: You can say 0 instead of 0rem, rest all is good!

**Review by BelideNikhil — APPROVED** [review]

**Review by MarkVed17 — APPROVED** [review]
Everything looks great, Nikhil! Your color palette is also nice.

**Comment by vijaytembugade** [comment]
> Everything looks great, Nikhil! Your color palette is also nice.

You better remember my name! 🤣🤣

**Review by saurabhchirde — APPROVED** [review]
Everything looks great to me, You can proceed to merge.

**Comment by preethasri** [comment]
Looking neat . Good to go

**Pull request merged**